maxim/tests/test_add_dataset_entries_integration.py

```
@@ -0,0 +1,441 @@
+        This test verifies that:
+        1. Dictionary entries can be successfully added to a dataset
+        2. Various data types (strings, numbers, objects) are handled correctly
+        3. Multiple attachments per entry are handled correctly (including real files)
+        4. Different file types (image, PDF, markdown) are uploaded successfully
+        5. The API response indicates success
+        """
+        print("\n📝 Testing dictionary input with real API (including 3 attachments: image, PDF, and markdown)...")
+        
+        # Verify files exist before creating attachments
+        assert os.path.exists("/Users/aryan/Desktop/maxim/maxim-py/files/free-nature-images.jpg"), "Image file not found"
+        assert os.path.exists("/Users/aryan/Desktop/maxim/maxim-py/files/Cursor – Models & Pricing.pdf"), "PDF file not found"
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)